js401codechallenge-day01/solution.js

```diff
@@ -0,0 +1,14 @@
+let values = [2, 56, 3, 41, 0, 4, 100, 23];
+
+let arrayFindings = function(arr){
+let array = arr;
```

Comment:
This doesnt hurt, but in the interest of DRY coding, you dont need to reassign an argument, because it has already been passed in.